tutor/src/components/task-plan/index.cjsx

```diff
@@ -56,9 +58,13 @@ EventShell = React.createClass
 PlanShell = React.createClass
   displayName: 'PlanShell'
 
```

Comment:
what do you think about this type of thing going into [match-by-router](https://github.com/openstax/tutor-js/blob/master/shared/src/helpers/match-by-router.cjsx) or something?  it seems like scrolling to top should be default behavior, and not scrolling should be optionally set.  or maybe that's just me missing the old router interface.